src/Paramore.Brighter.Extensions.Hosting/HostedServiceCollectionExtensions.cs

```diff
@@ -23,5 +23,19 @@ public static class HostedServiceCollectionExtensions
             brighterBuilder.Services.AddHostedService<TimedOutboxSweeper>();
             return brighterBuilder;
         }
+
+        public static IBrighterBuilder UseOutboxArchiver(this IBrighterBuilder brighterBuilder,
```

Comment:
I assume we think most folks will run this stand alone, perhaps with their sweeper?

Reply:
That's the plan, I'm looking at putting this in the same process as outbox sweeper so I've got an "Outbox management" service